cpp/src/io/parquet/experimental/hybrid_scan_helpers.cpp

```diff
@@ -261,6 +263,54 @@ aggregate_reader_metadata::select_payload_columns(
     valid_payload_columns, {}, include_index, strings_to_categorical, timestamp_type_id);
 }
 
+std::tuple<int64_t, cudf::size_type, std::vector<row_group_info>>
+aggregate_reader_metadata::select_row_groups(
```

Comment:
Reduced version of the same `select_row_groups()` function defined in raeder_impl_helpers.cpp